mods/ra/maps/soviet-01/map.yaml

```diff
@@ -625,10 +625,6 @@ Rules:
 			GenericStancePrefix: false
 			GenericVisibility: Enemy
 			ShowOwnerRow: false
-	^CivInfantry:
-		MustBeDestroyed:
-	^CivBuilding:
-		MustBeDestroyed:
```

Comment:
You need to keep this as it won't inherit it from ^Building like ^CivInfantry does from ^Infantry.